core/config/config.exs

```diff
@@ -142,6 +142,8 @@ config :core, BankingClient,
   certfile: "../banking_proxy/certs/client_certificate.pem",
   keyfile: "../banking_proxy/certs/client_key.pem"
 
+config :core, :leaderboard_enabled, false
```

Comment:
this can be moved/changed to:

config :core, :features, leaderboard: false

Currently the Next bundle has the following feature config:

config :core, :features,
  sign_in_with_apple: false,
  member_google_sign_in: false,
  password_sign_in: true,
  notification_mails: false,
  debug_expire_force: false